### PR TITLE
Updated Spotify to version 1.1.95.893.g6cf4d40c.

### PR DESCRIPTION
**Pull request by jedieaston**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/80581)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/s/Spotify/Spotify/1.1.95.893.g6cf4d40c/Spotify.Spotify.installer.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ Installers:
 - Architecture: x86
   Scope: user
   InstallerUrl: https://download.scdn.co/SpotifyFullSetup.exe
```
Comment (SpecterShell) [inline manifests/s/Spotify/Spotify/1.1.95.893.g6cf4d40c/Spotify.Spotify.installer.yaml:19]:
```suggestion
  InstallerUrl: https://upgrade.scdn.co/upgrade/client/win32-x86/spotify_installer-1.1.95.893.g6cf4d40c-39.exe
```

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by jedieaston** [comment]
@denelon can you assign this to someone for an override? spotify's been broken for a while :/

**Pull request merged**